src/NServiceBus.Core.Tests/Encryption/RijndaelEncryptionServiceTest.cs

```diff
@@ -28,23 +28,23 @@ public void Should_encrypt_and_decrypt()
         public void Should_encrypt_and_decrypt_for_expired_key()
         {
             var encryptionKey1 = Encoding.ASCII.GetBytes("gdDbqRpqdRbTs3mhdZh9qCaDaxJXl+e6");
+            var encryptionIV = Convert.FromBase64String("UjV8pJZ6ERroiFHluUlq6w==");
```

Comment:
Maybe I'm a bit too harsh but the line before it uses ASCII to bytes and then the second line uses Base64 string.